services/ai-navigator-app/0.2.8/defaults/cm.yaml

```diff
@@ -112,4 +90,4 @@ data:
     tolerations: []
 
     affinity: {}
-    kubectlImage: ${kubetoolsImageRepository:=bitnami/kubectl}:${kubetoolsImageTag:=1.29.6}
+    kubectlImage: ${kubetoolsImageRepository:=bitnami/kubectl}:${kubetoolsImageTag:=1.30.5}
```

Comment:
We should revert to the v1.29.6 bitnami image that ships with NKP 2.12.x

Reply:
I suspect `1.30` is backward compatible, but I suppose consistency is good.